### ISSUE TITLE
Monero Community Workgroup Meeting: October 12th 15:00 UTC

### ISSUE THREAD

**Issue by plowsof** [monero-project/meta#1088]
Location: [Libera.chat, #monero-community](https://libera.chat/) | [Matrix](https://matrix.to/#/#monero-community:monero.social?via=matrix.org&via=monero.social)

[Instructions for joining the monero.social Matrix server.](https://www.getmonero.org/resources/user-guides/join-monero-matrix.html)

Time
15:00 UTC [Check your timezone](https://www.timeanddate.com/worldclock/converter.html)

Moderator: plowsof

Please reach out in advance of the meeting if you would like to propose an agenda item.

Proposed Meeting Items:

1. Introduction
2. Greetings
3. Community highlights   
    - Kayabanerve shares a [summary thus far](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/449#note_26540) of FCMP related tasks. (Please report this fake kayaba twitter profile [x.com](https://x.com/kayabaNaerve)
    - Kraken delists XMR in Europe
    - [RINO wallet closing down, please withdraw funds](https://old.reddit.com/r/Monero/comments/1fwmhto/rino_wallet_closing_down_end_of_october/)
    - Featherwallet [2.7.0 tagged](https://github.com/feather-wallet/feather/releases/tag/2.7.0), join the [guix build/attest process](https://github.com/feather-wallet/feather/blob/master/contrib/guix/README.md) if you have spare resources and want to help
    - https://kewbit.org/ shares summary articles on FCMP/Carrot/Seraphis + more
    - https://pod.xmr.fan/ from pluja of kycnot.me
    - [monero.observer has gone independent! funding 'till 2026 is open](https://monero.observer/monero-observer-funding-2025/)
    -  [midipoet - Policy and Regulatory Framework Research](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/510) Funded independent of the CCS 
News: [Monero Observer](https://www.monero.observer/) - [Revuo Monero](https://revuo-xmr.com/) - [Monero Moon](https://www.themoneromoon.com/)    
5. [CCS updates](https://ccs.getmonero.org/)    [vtnerd](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/509)
  a. [Offline Signing Library for XmrSigner Production](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/495)    
  b. [Carrot animated video](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/506)    
  c. [SNeedlewoods-01_part-time dev work (1 month)](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/508)    
  d. [Add CypherStack Carrot Spec Review](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/511)      
6. Workgroup reports    
  a. Dev workgroup
  b. Localization workgroup
  c. Outreach workgroup
  d. Events workgroup - [MoneroKon 2024](https://github.com/MoneroKon/)
  e. Website workgroup
  f. Policy workgroup
  g. Research workgroup
7. Open ideas time    
8. Confirm next meeting date/time    

[Previous meeting including logs](https://github.com/monero-project/meta/issues/1083)    

Meeting logs will be posted here afterwards.

**Comment by plowsof** [comment]
Logs 
> __< plowsof >__ tomorrow is todays today, definitely      

> __< nioCat >__ I will not be able to attend     

> __< plowsof >__ nioCats +merge by proxy      

> __< plowsof >__ the meeting later today (just over 7 hours) https://github.com/monero-project/meta/issues/1088      

> __< plowsof >__ kewbit did you share your ccs update somewhere?     

> __< k​ewbit:matrix.org >__ No im really trying to get this tor PoW in docker compiled because there is no decent ones, I need to use it, It will help people test out the work so far, let’s see if I can finish in the next few hours     

> __< k​ewbit:matrix.org >__ It’s taking longer cause I’m doing the container multi-arch too 😂     

> __< plowsof >__ that's a fine update, thank you :D      

> __< 3​21bob321:monero.social >__ Updront payments is the new black?     

> __< 3​21bob321:monero.social >__ Upfront*     

> __< 3​21bob321:monero.social >__ Nioc your zoom ai clone will be able to attend meetings soon     

> __< midipoet >__ plowsof: you can remove my CCS from the agenda. I, gracefully, received an offer from a private entity to fund the work and so won't be going through the CCS anymore. I did want to go through the CCS (as i firmly believe it was a good proposal), but the offer of an upfront payment and removal of effort required to get the proposal through the CCS process was too good to refuse. Thanks all for the comments, criticism,      

> __< midipoet >__ and support.      

> __< n1oc >__ [CCS Proposals] midipoet closed merge request #510: midipoet - Policy and Regulatory Framework Research https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/510     

> __< plowsof >__ midipoet congratulations      

> __< s​yntheticbird:monero.social >__ congratulations midipoet.     

> __< k​ayabanerve:matrix.org >__ I have a CCS drafted and I'm just waiting on the final quote. I do have my own estimate (non-binding) for the range it'll be within however.     

> __< k​ayabanerve:matrix.org >__ Ideally, I'd be able to bring it up at the meeting today instead of kicking it down the road another two weeks     

> __< k​ayabanerve:matrix.org >__ (sorry to be so last minute with it, I only finished writing the draft last night)     

> __< s​yntheticbird:monero.social >__ sleep is important     

> __< 3​21bob321:monero.social >__ So 2kxmr is gone ?     

> __< k​ayabanerve:matrix.org >__ https://gist.github.com/kayabaNerve/3723d0a3f2b62ef8ef00c0c4a574fb8e for the draft which doesn't specify an amount due to the amount being pending. Obviously, a lot of commentary will depend on the amount (I can't off-hand ask for a million dollars last I checked) so I'll refrain from making the PR to the CCS repo until I have that info. All good if this can't be included this mee<clipped message>     

> __< k​ayabanerve:matrix.org >__ ting, I'll defer if there's time/sufficient basis to discuss to plowsof.     

> __< k​ayabanerve:matrix.org >__ SyntheticBird: Sleep isn't, energy is, and caffeine is an acceptable source of energy /s     

> __< midipoet >__ kayabanerve: maybe i missed this, but why the upfront immediate payment? Not saying it's necessarily bad, but just wondering why.      

> __< k​ayabanerve:matrix.org >__ midipoet: Because auditors generally take payment up-front, Cypher Stack is a trusted entity, and this policy has been done for Cypher Stack prior?     

> __< midipoet >__ Ah ok, fair enough.      

> __< k​ayabanerve:matrix.org >__ I'll clarify I wrote that in for the above reasons. I did not ask Cypher Stack if they'd be willing to have payment deferred and they did not explicitly raise this request.     

> __< midipoet >__ To be honest, i was mostly ignorant to the fact that auditors were paid up front.      

> __< k​ayabanerve:matrix.org >__ Eh. That may just be the auditors I've dealt with but I believe a few past CCSs to auditing companies (distinct from Cypher Stack) have also been paid upfront.     

> __< plowsof >__ Volatility buffers fan be reduced. This happened on one of CS prev ccs. Auditors have done work for us without 100% upfront, its just that CS have done alot of work already so less of a need to escrow      

> __< plowsof >__ s/fan/can*     

> __< k​ayabanerve:matrix.org >__ Veridise was paid after re: their work on divisors for FCMP++s. I'd actually have to double check my individual contracts with Cypher Stack to see if I paid before/on delivery 🤔     

> __< plowsof >__ The CCS gave 100% upfront to CS and we benefit ed from a volatility buffer removal      

> __< k​ayabanerve:matrix.org >__ The Bulletproofs+ audit seems to have been paid on delivery. For RandomX, the amount was paid as a lump sum on May 30, 2019. All of the funded audits in that CCS were dated July however.     

> __< k​ayabanerve:matrix.org >__ So it seems Kudelski/X41/Quarkslab took payment up front or tevador/hyc did and abstracted the payment schedule (no hate if they did, just noting I'm unclear the exact timeline as I try to look for am I right about this being general practice or if I'm wrong)     

> __< k​ayabanerve:matrix.org >__ Anyways, if it's controversial here, I can ask Diego to accept payment upon completion/ask for the quote to consider how it's payment upon completion.     

> __< o​frnxmr:monero.social >__ I don't think its general practice     

> __< o​frnxmr:monero.social >__ Its case-by-base basis, and depends on who what where when why how etc     

> __< o​frnxmr:monero.social >__ Proposer decides how theyd like to propose, and there may be auditors that wont accept xmr, and/or will need someone to convert the xmr for them.     

> __< o​frnxmr:monero.social >__ when we opened bp++ ccs, it was based on quotes. It was not determined to be paid up front and the amount changed after it was funded     

> __< o​frnxmr:monero.social >__ ccs for essential stuff doesnt follow standard rules, but is held to a higher standard. Example being that its entirely possible for generalfund to fully cover a ccs, to work with magic etc on payouts, and just as possible for payment to only happen once work is completed as it is for it to be paid at the start. It, at least in part, depends on the viability and quality of deliverables     

> __< s​yntheticbird:monero.social >__ We're still *hacking* the CCS because its where funds are but for god sake after FCMP++/Carrot/Jamtis is released it would be great time to generalizing it. For the sake of everyone (proposal, coordinators and donators)     

> __< k​ayabanerve:matrix.org >__ "generalizing it"?     

> __< s​yntheticbird:monero.social >__ Abstracting the CCS and identify common properties among proposal to make a general system that can formalize and cover all the corner cases that have recurrent for so long (upfront payment, rate, milestones, decisions, etc...)     

> __< k​ayabanerve:matrix.org >__ I mean, people just make arbitrary proposals now. While I'm not against updating the rules as helpful, I don't think it needs mass reform regarding the type of proposals which can be made.     

> __< k​ayabanerve:matrix.org >__ It arguably needs reform on how it's toxic to people who use it for pseudo-employment due to the latency and bureaucracy, yet at this point I'd suggest MAGIC hire developers to solve that.     

> __< k​ayabanerve:matrix.org >__ The issue is MAGIC doesn't have the funds on-hand and developers aren't eager to fundraise with MAGIC due to the reduced attention.     

> __< s​yntheticbird:monero.social >__ On another note: I think reddit bans greatly diminished our influence on social media. r/Monero was one of the most important community and I'm sure MAGIC would have received more attention (from users and whales) at the time.     

> __< midipoet >__ The problem with the CCS is how the community vets proposals     

> __< plowsof >__ kayabanerve this is the ccs where cs where paid upfront. no drama and they completed the work in a timely manner https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/443     

> __< midipoet >__ Vetting of proposals is, of course critical, but how it's done currently is problematic for a host of reasons.      

> __< k​ayabanerve:matrix.org >__ Yes, I'm aware of the precedent. I was trying to find arguments for/against it being general practice.     

> __< k​ayabanerve:matrix.org >__ midipoet: I haven't seen it as such except for the latency/bureaucracy. What issues do you have to raise?     

> __< plowsof >__ the CCS has sent funds directly to MAGIC once before. so that is an option     

> __< nioCat >__ <s​yntheticbird:monero.social> On another note: I think reddit bans greatly diminished our influence on social media.  <<>> as I am almost never there, could you explain what this banning is?     

> __< e​veroddandeven:monero.social >__ Hey     

> __< e​veroddandeven:monero.social >__ https://github.com/everoddandeven/monerod-gui     

> __< s​yntheticbird:monero.social >__ nioCat Reddit suddently started shadow-banning any users using VPN/Tor (even users using the reddit onion service) Meaning you can access Reddit, but people cannot see you (404), comments, upvotes and posts are invisible or automatically deleted. Naturally a lot of people that were using r/Monero were shadow banned and, therefore less comments, less posts. As of today I'm unable t<clipped messa     

> __< s​yntheticbird:monero.social >__ o inform the reddit community on Cuprate advancements. I would need to buy a residential proxy for this which is not free and sometimes don't work.     

> __< e​veroddandeven:monero.social >__ Take a look on my monerod gui, I have done a lot of work since last time     

> __< s​yntheticbird:monero.social >__ LETS GO     

> __< midipoet >__ kayabanerve: i think we (me included) should try and appraise proposals against "does this make enough sense (timing, milestones, cost, etc) that someone (we have lots of different types of people in the community) might be willing to donate to it", as opposed to "am i going to donate to this?". The former is a more rational approach, attempting to use some relatively objective and unbiased measures. The latter is less      

> __< midipoet >__ so. For example, the proposal i recently submitted raised valid concerns from the community on scope, domain/topic, value, cost, etc, but it also instigated a different sort of discussion that wasn't helpful for anyone. In the end it was just easier for me to remove myself from the process. I also don't think it's productive, at all, to point to previously successfully completed proposals as evidence that future      

> __< midipoet >__ proposals, or opinions should be disregarded. This seems wholly non-sensical. One would think successfully completed proposals should be viewed positively by the community (at best), or (at least) as indication that others may value slightly different work/outputs than others. That's my two cents, i am sure not everyone agrees though.     

> __< e​veroddandeven:monero.social >__ builds are working on windows, mac and linux. There are still some bugs that I need to fix, but it is already 90% of the way there. If you have any suggestions, please let me know. (Sorry for the spam=     

> __< e​veroddandeven:monero.social >__ builds are working on windows, mac and linux. There are still some bugs that I need to fix, but it is already 90% of the way there. If you have any suggestions, please let me know. (Sorry for the spam)     

> __< s​yntheticbird:monero.social >__ everoddandeven:  thanks for you job will try it later in the day.     

> __< k​ayabanerve:matrix.org >__ I advocate for any proposal I find reasonable, not for any proposal I care about.     

> __< s​yntheticbird:monero.social >__ midipoet I agree with you and I'm everyone. So everyone agrees with you.     

> __< s​yntheticbird:monero.social >__ Naturally, in order to judge a proposal many reviewers will first project their personal expectations and values on it in order to find concerns or/and interests. I think it's perfectly normal. However, some really seems to value themselves as representing the average donators. Add to that the fact that we don't really know exactly how a decision is taken (plowsof? community + plo<clipped messa     

> __< s​yntheticbird:monero.social >__ wsof? community + plowsof + luigi?) and the proposer is just found trapped in an endless, *Can we just try it at least?*. Regarding your proposal, I agrees with rbrunner view on it and I found it good and with potential benefits, but was just not convinced by the hour rate. Tho it's my concern, I  don't expect (and don't want) it to be a blocker. Just raising the question to more <clipped messa     

> __< s​yntheticbird:monero.social >__ people (including you), *Is it sound? 90$/hr for that seems like beyond the chart*.     

> __< s​yntheticbird:monero.social >__ And I think it should be nothing more than questions to ask for when the meeting will take place. Instead, at the moment, at each f*cking question comes a potential drama and stress to for the proposer.     

> __< o​frnxmr:monero.social >__ How is a decision taken? In reality? Or ideally?     

> __< o​frnxmr:monero.social >__ Ideally, a combination of weighted consensus     

> __< s​yntheticbird:monero.social >__ well maybe reality have wrong weights then.     

> __< o​frnxmr:monero.social >__ Reality, luigi is 49-51%     

> __< s​yntheticbird:monero.social >__ here we go     

> __< plowsof >__ don't save your questions until the meeting, put them under the porposal as soon as possible. quick fire questions in a meeting (where we are generally rushed for time)  is not ideal      

> __< s​yntheticbird:monero.social >__ plowsof ack.     

> __< o​frnxmr:monero.social >__ Proposals can have full community support and still be veto'd     

> __< o​frnxmr:monero.social >__ Or can be straight up scams with sockpuppets voting for them, and be merged     

> __< o​frnxmr:monero.social >__ Or can be monero tesla     

> __< o​frnxmr:monero.social >__ Plowsof exists to try to make the process based on sane consensus, but he's powerless to block or merge anything. He coordinates, in part, try to get community to speak up "or forever hold their peace"     

> __< plowsof >__ syntheticbird depends on the question of course*       

> __< o​frnxmr:monero.social >__ disclaimer: I am speaking on behalf of plowsof w/o his input     

> __< s​yntheticbird:monero.social >__ Yes thats why I said *potential drama*.     

> __< vthor >__ "And I think it should be nothing more than questions to ask for when the meeting will take place. Instead, at the moment, at each f*cking question comes a potential drama and stress to for the proposer." <- this. But even worse is for me people not even reading the response, simply answering stuff to promote their stuff, and asked have you even read my answer replying not even read your answers here. How can one intepret that different then simply as      

> __< vthor >__ trying to do everything to stop it?     

> __< s​yntheticbird:monero.social >__ for real.     

> __< s​yntheticbird:monero.social >__ complete agree     

> __< vthor >__ The other thing for me, what disturbs me, but this can be only a very personal issue, because I end up always to make more work and charge less. But it seams to me whatever rate you will call, it is too much, but from people who are not intrested into your proposal and and probably not even onces donated anything to any proposal. Like monero(money) would be finite and better to don't pay anything to anybody because the faucet will close. This is for me      

> __< vthor >__ crazy, shouldn't monero prosper people in taking the state(s) out of the equation? Money(or monero in this case) get generated by value, that means somebody creates value and this will reflect the value in xmr, not the other way around. I personal think we should abolish as far as possible xmr from fiat and xmr, but this doesn't happen if one calculate from fiat into xmr. I decided for me that I want to charge for a normal promodoro session (100min      

> __< vthor >__ highly focused work) 1 xmr here and 2 xmr for closed source work. Reasoning behind is easy - you don't work only in this sessions where you produce, you also research, build mock ups, test code (you would normally show never anybody, because it's only a hack to see the direction you are going. But then you see all the drama from that gatekeepers, and I personally can not take any drama, so I go down with my rate, but for sure at the end you arrive by 2-     

> __< vthor >__ 3x the time estimated, if you have no drmatic findings on the way. But then for sure the rate fiat/xmr goes up just then and you adjust again down - not because you think it would be right or fair, but avoid at least one drama, and at the end when you get paid the rate will be lower then on the starting point. I mean, for me it is crazy. When there is nobody for whom the value is a good deal, there will be no funding, so why should a person not donating      

> __< vthor >__ telling a person donating what value is okay?     

> __< o​frnxmr:monero.social >__ donor are anonymous.     

> __< o​frnxmr:monero.social >__ Nobody commenting has to do so with a disclaimer "i am donating". And donating to a proposal doesn't add weight to your commentary     

> __< o​frnxmr:monero.social >__ edit: donor are allowed to be anonymous.     

> __< vthor >__ Heck, there in the wall of text are more then one issue wrapped in. But mostly I think it breaks down to two things. 1) I think the donators are able enough to see if the agree with the value. 2) the thinking in fiat (although it is still afacting one) is contra the aim of monero IMO.     

> __< o​frnxmr:monero.social >__ 2 = i agree     

> __< o​frnxmr:monero.social >__ some proposals are made in $, because the funding is for outside parties (such as audits)     

> __< vthor >__ " donor are allowed to be anonymous." <- what does it matter, the donor is the most honest voter, he puts the money where his mouth is. This is the point IMO. I can get my CCS merged, but if it is not worth anything for a donor, then there is no funding. All the people critizizing everybody for the rates, will not then throw something in the pot. This is what I mean with it.     

> __< o​frnxmr:monero.social >__ aside from those, i don't like that ppl want to add volatility buffers, or abandon work when price dips (but never refund $ when price moons)     

> __< o​frnxmr:monero.social >__ Some ppl add expiry date to their funding proposal. Such as "if not funded in 2 weeks, close it and donate money to jet fund"     

> __< o​frnxmr:monero.social >__ One problem with yolo merging everything, is having proposals sit unfunded forever     

> __< o​frnxmr:monero.social >__ Look at rottenwheel's "pay me 5k for being scammed" kuno. If we had that on the "funding" page, it just looks bad.     

> __< vthor >__ I think the optimal curve for xmr would be a just a bit over the real inflation (this would lock in users into xmr and weaken fiat and hopefully lead to a colapse of the state so everybody can propser on ther fruits they are creating instead ending up us slaves). But when here seems to be more a socialist (gatekeeper that nobody earns "too much") then it sends IMO a wrong signal.     

> __< o​frnxmr:monero.social >__ Then you also need to look at history of completed and in-progress ccs. What ccs have been completed successfully where the work is live/active today? What ccs are dead?     

> __< o​frnxmr:monero.social >__ We have a _very_ low success rate when ccs aren't directly involving core development     

> __< vthor >__ "Then you also need to look at history of completed and in-progress ccs. What ccs have been completed successfully where the work is live/active today? What ccs are dead?" <- don't agree, I full blown my privacy for the first CCS I made. But in reality, the payouts are after the milestones. If the payout would be before I could agree.     

> __< o​frnxmr:monero.social >__ for successed, heres an easy one: Feather wallet     

> __< o​frnxmr:monero.social >__ If xmrsigner finished? Usable? Live?     

> __< o​frnxmr:monero.social >__ Is*     

> __< vthor >__ "We have a _very_ low success rate when ccs aren't directly involving core development" <- I think that is mostly because of drama.     

> __< o​frnxmr:monero.social >__ No, its because the ccs's were no-good     

> __< vthor >__ "If xmrsigner finished? Usable? Live?" finished: yes, usable: yes, but I would not recommend it. Live? Don't know what is live for you, if it is production state, in my opinion not, but if you would see it from the standpoint of the original MoneroSigner proposal I would say it is overdelivered.     

> __< vthor >__ "No, its because the ccs's were no-good" -> elaborate.     

> __< o​frnxmr:monero.social >__ Thats saying thr original monerosigner proposal was no-good, yeah?     

> __< o​frnxmr:monero.social >__ And not production rdy = state of every other non-core proposal     

> __< o​frnxmr:monero.social >__ Everything in left incomplete or in an unusable state, abandoned or worse     

> __< vthor >__ Really? Didn't you mention feather?     

> __< o​frnxmr:monero.social >__ i said feather was a success     

> __< vthor >__ "Everything in left incomplete or in an unusable state, abandoned or worse" <- but I think that comes together with that drama/     

> __< o​frnxmr:monero.social >__ One of the few     

> __< vthor >__ "i said feather was a success" " And not production rdy = state of every other non-core proposal", is feather core?     

> __< o​frnxmr:monero.social >__ drama isnt a milestone     

> __< o​frnxmr:monero.social >__ "We have a _very_ low success rate when ccs aren't directly involving core development" << very low     

> __< o​frnxmr:monero.social >__ "for successed, heres an easy one: Feather wallet" << feather one of the very low exceptions     

> __< vthor >__ "drama isnt a milestone" seems I didn't express myself well. I think that a lot get abandoned because of the drama... Not sure, but my impression.     

> __< o​frnxmr:monero.social >__ No. a lot of Ppl do 2/3 milestones then ride off into the sunset     

> __< vthor >__ "for successed, heres an easy one: Feather wallet" << feather one of the very low exceptions, ah, got it now. Thanks for the patience to clarify.     

> __< vthor >__ "No. a lot of Ppl do 2/3 milestones then ride off into the sunset" <- but what is the reason for?     

> __< o​frnxmr:monero.social >__ Or scope the project to be incomplete, then ride off into the sunset     

> __< o​frnxmr:monero.social >__ Reason is $. It (was) very easy to get paid to not do you work herr     

> __< vthor >__ I guess it is underfunding and lack of in the project. Well, maybe some bad apples, but I don't think that is the majority.     

> __< o​frnxmr:monero.social >__ Devs used to roll ccs for years on end and never fix a single bug     

> __< o​frnxmr:monero.social >__ It was the majority b4 plowsof ahowed up with his puppy     

> __< vthor >__ "Devs used to roll ccs for years on end and never fix a single bug", you mean after CCS finished?     

> __< o​frnxmr:monero.social >__ and there are some ppl still here who still miss the old days when you could get funding to act like youre a good person, meanwhile you were making 20k off of monerokon     

> __< o​frnxmr:monero.social >__ I mean, devs on ccs used to open a new ccs every 3 months and never do any work     

> __< o​frnxmr:monero.social >__ Community would vote for them w/o checking any work, and other devs wouldnt speak up _because they dont want drama_     

> __< vthor >__ weird     

> __< o​frnxmr:monero.social >__ after one of these devs was exposed by perfect-daemon, he threatened to sue everyone     

> __< vthor >__ sue? WTF     

> __< o​frnxmr:monero.social >__ Threatened to shoot me     

> __< o​frnxmr:monero.social >__ Went on a reddit campaign to cancel selsta and luigi     

> __< vthor >__ WTF, what shit people are there in the world.     

> __< o​frnxmr:monero.social >__ scammers arent good ppl, and theres a reason why good ppl dont speak up     

> __< vthor >__ :o can I express myself here even like that or issue with libera?     

> __< o​frnxmr:monero.social >__ bad actors are good at acting     

> __< o​frnxmr:monero.social >__ i dont think youre crossing any boundaries     

> __< vthor >__ but they end normally up in government :D So normally they have a title or a badge where you can identify them (only half serious)     

> __< o​frnxmr:monero.social >__ you have people like erc, who misrepresented haveno, who gambled away the funds, and then went to reddit and twitter to try to blame woodser and attack selsta     

> __< vthor >__ "i dont think youre crossing any boundaries" <- thank you, I'm never sure about that, normally my accounts get everywhere blocked when I voice my opinion in the last years, I'm already even afraid to write email :D (not really, but almost).     

> __< o​frnxmr:monero.social >__ i don't think good proposals see too much unnecessary drama. I think the major part is helping set a proposal up for success     

> __< vthor >__ :D reddit I got also blocked for being honest, reddit looks for me like a platform of very leftist wokies hostile egotrip insanes. I deleted all my comments and account after getting blocked once. And with twitter, but I think social media in general, most people have time for this crap, so you will ever here the side from the less productive more, so I guess it goes at the end in the same direction.     

> __< o​frnxmr:monero.social >__ Just like a pull request. Just because the author thinks its perfect, doesnt mean that more eyes can't find issues or help improve it. and "more" isnt quantity. Nobody needs useless trolls reviewing stuff     

> __< midipoet >__ Who made 20k from MoneroKon?     

> __< o​frnxmr:monero.social >__ Msvb     

> __< z​adocta:matrix.org >__ Wokies ? Leftists ? WTF     

> __< z​adocta:matrix.org >__ Try Feminists     

> __< o​frnxmr:monero.social >__ 9k merch, 6k firo, other invoices that i dont rememeber     

> __< vthor >__ "i don't think good proposals see too much unnecessary drama. I think the major part is helping set a proposal up for success" <- don't know. How you set up a proposal for success?     

> __< o​frnxmr:monero.social >__ By checking it for issues, overselling, underdelivering etc     

> __< vthor >__ "Just like a pull request. Just because the author thinks its perfect, doesnt mean that more eyes can't find issues or help improve it." <- Ack. For me it was weird that there was so many people cheering on X, but I get (almost) no feedback.     

> __< midipoet >__ ofrnxmr: i am not going to get into this with you again, but from someone on the inside of nearly all physical events we have done, msvb was always one of the most committed and hardest workers. They (a number of times) supplied things that nobody else seemed able to, and those things benefitted Monero and it's ecosystem. I would even go as far as to say the Monero related events have been lessened by the fact we don't      

> __< midipoet >__ have the same level of effort and commitment of msvb anymore.      

> __< vthor >__ "Try Feminists" <- ?     

> __< midipoet >__ ofrnxmr: the issue is that you seem completely unable to see others perspective, and continually bully people until they leave.     

> __< midipoet >__ At least that seems your intention      

> __< o​frnxmr:monero.social >__ effort and commitment have NOTHINGZ TO DO WITH MERCH OR BADGES     

> __< o​frnxmr:monero.social >__ I dont bully ppl til they leave, i ask them to leave     

> __< midipoet >__ ofrnxmr: in some instances, monero events would have had no merch and no badges without msvb     

> __< o​frnxmr:monero.social >__ good     

> __< midipoet >__ Because nobody else stepped up to source/manufacturer them     

> __< o​frnxmr:monero.social >__ Badges were a scam and he stole a sponsor with them     

> __< o​frnxmr:monero.social >__ Then lied about using hardware⊙go to sell them     

> __< o​frnxmr:monero.social >__ And lied about selling them at defcon     

> __< midipoet >__ Some of the most prized monero "objects" came from msvb and his "design lab" or whatever you call it      

> __< o​frnxmr:monero.social >__ And tried to sell them TO ccs at a 100% markup     

> __< midipoet >__ ofrnxmr: i am aware of your perspective      

> __< o​frnxmr:monero.social >__ Scammets gonna scam and birds of a feather, midi     

> __< o​frnxmr:monero.social >__ I already said you guys stick together. I didnt kick him out of mk, you guys did     

> __< midipoet >__ ofrnxmr: do you know how many badges msvb has produced for Monero?     

> __< midipoet >__ Have you even ever seen them?     

> __< o​frnxmr:monero.social >__ for monero? 0     

> __< o​frnxmr:monero.social >__ 160euros he was asking for them at mk     

> __< midipoet >__ Ok, for monero events..     

> __< o​frnxmr:monero.social >__ ceetee is the badges expert     

> __< o​frnxmr:monero.social >__ iirc they are generic, missing features, and are reused for multiple currencies and events     

> __< midipoet >__ So you don't know?     

> __< o​frnxmr:monero.social >__ How many? Over the years? No, i dont keep track     

> __< nioCat >__ there were different badges, I got one for free and gave it away for cost     

> __< o​frnxmr:monero.social >__ It was like 200 badges that firo purchased at $60/badge     

> __< o​frnxmr:monero.social >__ 100*     

> __< o​frnxmr:monero.social >__ and i dont remember how mant he wanted ccs to purchase at $40 a pop     

> __< o​frnxmr:monero.social >__ And i dont know what he told vik the price was in email from hardware⊙go     

> __< nioCat >__ the issue with msvb was that when anyone questioned anything he ignored them and I'm not just talking ofrn     

> __< o​frnxmr:monero.social >__ Cryptohip badges     

> __< o​frnxmr:monero.social >__ 20000 USD     

> __< o​frnxmr:monero.social >__ Cost of four hundred electronic badges     

> __< midipoet >__ I'd ignore people in here as well, if i had more self discipline      

> __< o​frnxmr:monero.social >__ https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/397     

> __< nioCat >__ anyway I promised to miss today's meeting and I keep my promises     

> __< nioCat >__ .bbl     

> __< nioCat >__ move carrot audit to funding required and change the name of FCMP++ to rabbit      

> __< o​frnxmr:monero.social >__ Moment of silence b4 the meeting     

> __< vthor >__ "I'd ignore people in here as well, if i had more self discipline" <-  :D can feel you, try also every day to ignore so much and I can't mostly.     

> __< r​ottenwheel:kernal.eu >__ What are we fighting about today?     

> __< plowsof >__ msvb still has the best quality hats / tshirts. they have never been matched to this day     

> __< r​ottenwheel:kernal.eu >__ It must be the beans...     

> __< plowsof >__ so im told !     

> __< vthor >__ No more beans please, they here everywhere     

> __< r​ottenwheel:kernal.eu >__ Lol.     

> __< plowsof >__ Meeting time https://github.com/monero-project/meta/issues/1088     

> __< r​ottenwheel:kernal.eu >__ Imagine conquering half the world, have tons of different food options to pick from and of all them, the absolute lovely British people had to pick two things!     

> __< r​ottenwheel:kernal.eu >__ Toast and beans. 😂😂     

> __< vthor >__ :D     

> __< plowsof >__ we exported cricket, thats the best we could offer      

> __< plowsof >__ greetings everyone      

> __< r​ottenwheel:kernal.eu >__ Oy mate! It is meeting time!     

> __< vthor >__ hey plowsof :)     

> __< r​ottenwheel:kernal.eu >__ Ya got a loicense for that meeting m8?     

> __< plowsof >__ danbob was asking about the 2000 xmr here earlier :     - Kayabanerve shares a [summary thus far](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/449#note_26540) of FCMP related tasks. (Please report this fake kayaba twitter profile [x.com](https://x.com/kayabaNaerve)     

> __< s​needlewoods:monero.social >__ hello     

> __< k​ayabanerve:matrix.org >__ Oh, that's what that was about?     

> __< plowsof >__ kayabanerves fake clone twtiter account is going strong haha     

> __< k​ayabanerve:matrix.org >__ I saw a message asking about 2000 XMR and had no idea the context.     

> __< plowsof >__ yes i assume it was the research fund     

> __< f​alcon8:matrix.org >__ I wish I had fake profiles under my name :P     

> __< r​ottenwheel:kernal.eu >__ xenu pinche vato culero.     

> __< k​ayabanerve:matrix.org >__ The 2000 XMR for FCMP++ research (if that is the exact amount raised) isn't gone. It still has the majority of itself held by the CCS. I'm making a new CCS to audit monero-serai/monero-wallet. That's a distinct scope from the FCMP++ tasks and isn't dipping into that pool of funds.     

> __< plowsof >__ i have made the first purchase from plowsofs fund today, i purchased a microwave using a monero funded giftcard. it was 59 euros. i hope the community approves     

> __< k​ayabanerve:matrix.org >__ I'm pretty sure that is the amount we raised for the FCMP++ Research task, I just have to double check before I comment with certainty.     

> __< r​ottenwheel:kernal.eu >__ Thievery!     

> __< r​ottenwheel:kernal.eu >__ plowsof resign and refund immediately!     

> __< plowsof >__ :'(      

> __< plowsof >__ thanks for clarifying kayabanerve      

> __< plowsof >__ https://pod.xmr.fan/ from pluja of kycnot.me if keeping up with the releases :)     

> __< plowsof >__ new featherwallet is out https://featherwallet.org/changelog/      

> __< vthor >__ "i have made the first purchase from plowsofs fund today, i purchased a microwave using a monero funded giftcard. it was 59 euros. i hope the community approves" <- I highly disaprove. :D Don't kill yourself plowsof, we need you, microwave ovens are evil. Use an air fryer is much more healthy.     

> __< plowsof >__ rottenwheel has joined in with verifying the build hashes      

> __< plowsof >__ 💗thank you for caring vthor , i wanted to switch from an icecream based diet to 'warm food' diet       

> __< ofrnxmr >__ buy a bbq     

> __< plowsof >__ congratulations to midipoet for getting private funding for his CCS proposal      

> __< r​ottenwheel:kernal.eu >__ plowsof yeah, let me know if there are other gitian-like builds around I could help check in the future. Feather one was fun!     

> __< plowsof >__ monero observer is busy fund raising for the own services https://monero.observer/monero-observer-funding-2025/     

> __< r​ottenwheel:kernal.eu >__ Vosto is CIA, be careful!     

> __< ofrnxmr >__ Thats why its called a build party, rotten. If it wasnt fun, it would just be called a build     

> __< ofrnxmr >__ Good on observer. I always said that newsletters should strive to maintain funding via sponsorships and readership     

> __< r​ottenwheel:kernal.eu >__ Anyone talking? Only hear a zzz, like a mosquito buzzing around.     

> __< plowsof >__ ofrnxmr is overseeing basicswapdex progress. so far ive made 2 payouts from their bounty wallet to fund BCH<>XMR atomic swaps being added to that platform      

> __< plowsof >__ https://kewbit.org/  is about to release a big update for their Haveno multi os app but not quite yet      

> __< ofrnxmr >__ No, just typing, youre hearing things again     

> __< ofrnxmr >__ Bch's xmr swap protocol integration should be done and released by end of november     

> __< r​ottenwheel:kernal.eu >__ kewbit has been writing more than a New York Times editor!     

> __< r​ottenwheel:kernal.eu >__ He's the new LocalMonero Knowledge section!     

> __< r​ottenwheel:kernal.eu >__ Guy has nothing better to do with his life than work on Haveno Plus full-time and now write explanatory articles!     

> __< ofrnxmr >__ Bsx should have a new release hopefully tomorrow     

> __< plowsof >__ oh yes, i changed a number 2 to a number 4 for monero core repo CI runners. now the windows runner gets 4 threads instead of 2! and added caching to monero-gui docker builds. can you imagine all the devs waiting for workflows to complete instead of improving monero? things are going to be wild now (/s)      

> __< plowsof >__ nice ofrnxmr      

> __< r​ottenwheel:kernal.eu >__ Right... There are hundreds of thousands willing to work on wallet2...     

> __< plowsof >__ another reminder about RINO closing down (https://old.reddit.com/r/Monero/comments/1fwmhto/rino_wallet_closing_down_end_of_october/)     

> __< ofrnxmr >__ also an important mention is that the market maker script has an important new feature that makes providing liquidity much, much easier      

> __< ofrnxmr >__ s/has/will be in new release. Will announce on X @basicswapdex      

> __< vthor >__ "Right... There are hundreds of thousands willing to work on wallet2..." <- I'm willing, if it shrinks instead of growing :D (and myself made a PR to add more methods :S )     

> __< plowsof >__ the tweaks to runners / cache came about from me pinning / mirroring all the inputs for gui's dockerfile , which should be ready for a 'WIP' PR soon. in an effort to help improve gui's rating at https://monerosecurity.org/ , an initiative tobtoht started      

> __< plowsof >__ not to improve guis rating* to show the differences of wallets in the ecosystem      

> __< k​ayabanerve:matrix.org >__ Shill, yet my upcoming CCS does have increased notability since RINO is closing down.     

> __< plowsof >__ looking forward to discussing or seeing that proposal      

> __< plowsof >__ anything else people want to raise?     

> __< ofrnxmr >__ Your multisig implementation?     

> __< k​ayabanerve:matrix.org >__ I linked the draft plowsof     

> __< k​ayabanerve:matrix.org >__ I was hoping it could at least be initially discussed today.     

> __< plowsof >__ sorry, moment      

> __< k​ayabanerve:matrix.org >__ I don't have the final amount. It should be on the scale of 750-1000 XMR.     

> __< plowsof >__ i sen the gist, and i got distracted thinking about a gist related joke someoneelse made'      

> __< plowsof >__ this is the linked draft: https://gist.github.com/kayabaNerve/3723d0a3f2b62ef8ef00c0c4a574fb8e     

> __< k​ayabanerve:matrix.org >__ Which is... nontrivial. It's also to audit a reimplementation of the transaction protocol, provide a high-level embeddable wallet/multisig derived from strong academia, and it's not something I've taken money for prior (yet have spent money on prior).     

> __< ofrnxmr >__ Is cuprate using monero-serai     

> __< ofrnxmr >__ (asking for the meeting logs)     

> __< k​ayabanerve:matrix.org >__ It's also used by Cuprate. Since Cuprate has been deemed worth funding via the CCS, this should be amenable as if it didn't exist, Cuprate would've had to independently build monero-serai which took me years?     

> __< k​ayabanerve:matrix.org >__ It's also used by Serai, all the justifications that provides.     

> __< k​ayabanerve:matrix.org >__ And multiple wallets/app have expressed interest     

> __< k​ayabanerve:matrix.org >__ And multiple wallets/apps have expressed interest     

> __< k​ewbit:matrix.org >__ I’m aiming to have the best Dockerized tor in the world by the end of the day, I’m gunna blog about it 😂     

> __< k​ayabanerve:matrix.org >__ I'll make the CCS proper once I have the amount finalized. I'd hope this can be agreed to be merged at the next meeting in two weeks. That's why I posted the draft before this meeting to try and solicit the necessary initial comments.     

> __< k​ayabanerve:matrix.org >__ It's all good if that timeline doesn't work out however. It's my goal, not my expectation.     

> __< ofrnxmr >__ MRL is the best place to get consensus for this @kaya      

> __< plowsof >__ tanks for sharing the draft early kayabanerve. generally ew defer to MRL/NWLB/MoneroTech meetings for audits and such e.g. if the scope is acceptable or could incorporate other things. this is quite different though, but still      

> __< k​ayabanerve:matrix.org >__ I wouldn't have considered it related to research.     

> __< k​ayabanerve:matrix.org >__ I do understand getting their comment due to the multisig aspects, yet the lib itself is largely dev.     

> __< k​ayabanerve:matrix.org >__ "MoneroTech"?     

> __< plowsof >__ i dont know, its possible someone says 'hey can you cover this thing that will benefit this too?' i really have no idea      

> __< k​ayabanerve:matrix.org >__ TBC, I am fine getting it on the next MRL agenda :) this feedback on process is great to have from this meeting (much better than getting it two weeks from now)     

> __< s​needlewoods:monero.social >__ MoneroTech is the new proposed name for what's known as NWLB     

> __< ofrnxmr >__ monerotech = monero-dev-lab, NWLB     

> __< plowsof >__ 750-1000 for a large audit from a trusted auditor (cypherstack) who has done work for the project over the past decade wouldnt cause much drama i should think. the funds release immediately upfront could.      

> __< r​ucknium:monero.social >__ I completed Milestone 1 of my statistical research CCS. Work products: defenses against black marble flooding attacks, the privacy of Monero's peer-to-peer network, and the safety of adjusting the 10 block lock: https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/439#note_26679     

> __< plowsof >__ 👏     

> __< ofrnxmr >__ also, stressnet is over for now     

> __< ofrnxmr >__ Ruck and spackle did most of the work stressing out my node     

> __< k​ayabanerve:matrix.org >__ This isn't next-gen so it shouldn't be NWLB IMO. If people here also want me to discuss it there (and not just MRL), I'll bring it up there too.     

> __< plowsof >__ true      

> __< k​ayabanerve:matrix.org >__ I'll reiterate that estimate isn't guaranteed. It could be less, could be more, and I'm waiting on the final quote.     

> __< plowsof >__ now unless there are other things we can move onto the ccs ideas      

> __< plowsof >__   a. [Offline Signing Library for XmrSigner Production](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/495)         

> __< ofrnxmr >__ What's this look like to get to production quality      

> __< vthor >__ "What's this look like to get to production quality " <- is a question?     

> __< ofrnxmr >__ Yes     

> __< plowsof >__ fans of xmrsigner need this so they dont have monero-wallet-rpc running on a hardware restricted device      

> __< ofrnxmr >__ Sorry     

> __< plowsof >__ Rucknium mentioned the previous/open ccs bringing it to production quality     

> __< vthor >__ It will be based on the buildroot2 environment, with the libarary compiled in the environment, without the RPC     

> __< ofrnxmr >__ Right. So i'm wondering how many ccs will it take til this is finished     

> __< plowsof >__ what can we do with your xmrsigner project now vthor? can we send/receive monero offline? sign , send key images via qr codes using the xmrsigner?     

> __< vthor >__ Yes, all of it, but because of the wallet RPC (seem loock ahead makes it so slow, but couldn't investigate it because I was no functioning for some days) but it is working only on Pi OS     

> __< plowsof >__ and i see tobtoht has left feedback which you responded to 3 weeks ago https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/495#note_26230 , have you incorporated his feedback into your proposal?     

> __< vthor >__ what IMO is not a good idea for a production device.     

> __< vthor >__ "and i see tobtoht has left feedback which you responded to 3 weeks ago https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/495#note_26230 , have you incorporated his feedback into your proposal?" <- not yet, I was to slow in the morning. I have a draft, but not nothing to share on the rework of the CCS.     

> __< ofrnxmr >__ Lookahead is recommended to be lowered on hw wallets     

> __< ofrnxmr >__ https://docs.getmonero.org/interacting/monero-wallet-cli-reference/?h=lookahead#restore-wallet     

> __< vthor >__ So for production, there need to be the library and buildroot and the library build in buildroot itself instead coping binaries and libs into the image.     

> __< plowsof >__ the C ABI proof of concept linked in the proposal gained sneurlax's attention https://github.com/DiosDelRayo/c_abi_for_cpp_code_PoC     

> __< sneurlax >__ it hasn’t caught my attention but has now, thanks!     

> __< ofrnxmr >__ Modifying* lookahead is recommend*     

> __< ofrnxmr >__ "It will take up to 25 minutes with default settings. This is because hardware devices are slow to pre-generate subaddresses. To mitigate use low --subaddress-lookahead 5:20."     

> __< vthor >__ "Lookahead is recommended to be lowered on hw wallets" <- not 100% sure, but I think the cold wallet needs no lookahead at al, because view only wallet provides the outputs, and hot wallet should only deliver or this outputs the key images and finally sign the transaction. the whole RPC way is uterly wrong and was only done to not cancel the previous CCS.     

> __< plowsof >__ vthor if this proposal is successful, and 240 hours was spent completing it. what do we get at the end of this? it can be a brief answer as its short notice      

> __< plowsof >__ everyone who showed support for monero on a seedsigner device would want this proposal i feel, the UX currently is , well, slow , i would hold off on this for now      

> __< vthor >__ "https://docs.getmonero.org/interacting/monero-wallet-cli-reference/?h=lookahead#restore-wallet" <- thank you for the link. Will check out if RPC understands the same argument, I was about to search in the source to try to got to 0.     

> __< ofrnxmr >__ iirc it does not     

> __< ofrnxmr >__ There is an open issue about it     

> __< ofrnxmr >__ https://github.com/monero-project/monero/issues/8954     

> __< plowsof >__ i would like to skip the Carrot animated video and move onto sneedlewoods as they are here, unless vrhor wants to continue      

> __< vthor >__ "vthor if this proposal is successful, and 240 hours was spent completing it. what do we get at the end of this? it can be a brief answer as its short notice " A production ready (buildroot2) xmr signer, a library useable for any signing device with documentation and quick start for the next fellow developer, and the answer if there could be a bare metal XmrSigner on Pi Zero (from the library side - still cam side would be needed to be tested)     

> __< plowsof >__ nice      

> __< vthor >__ "There is an open issue about it" - thank you, then I will try to test is changing the source. But anyway RPC will never hit buildroot, and IMO it's not worth the time to get RPC running on buildroot     

> __< p​lowsof:matrix.org >__ c. [SNeedlewoods-01_part-time dev work (1 month)](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/508)     

> __< ofrnxmr >__ No brainer, my only q is - are we even doing seraphis anymore?      

> __< sneurlax >__ sorry if this question is out of place, but that DiosDelRayo/c_abi_for_cpp_code_PoC is interesting but I’m also looking for any additional projects which build any part of monero as a lib (idc if it’s dynamic or static) if anyone has any references     

> __< vthor >__ This lib in my CCS will build a static library for the hot wallet part of the offline signing process.     

> __< sneurlax >__ seraphis addresses may yet be integrated after carrot aiui.  but personally I like that the address format change was deferred behind what I see as more important changes (FCMPs, making a view key that can discern spends)     

> __< p​lowsof:matrix.org >__ rbrunner has left their support for sneedlewoods' proposal. SW has been working with the NWLB group. the proposal is for,, 2.15xmr     

> __< k​ayabanerve:matrix.org >__ ofrnxmr: no, but yes.     

> __< k​ayabanerve:matrix.org >__ Your confusion is wrong and shouldn't exist yet perfectly valid.     

> __< ofrnxmr >__ awesome     

> __< k​ayabanerve:matrix.org >__ I don't see us doing, and won't advocate for, Seraphis, the migration and new output key/linking tag definition cryptographically defined.     

> __< sneurlax >__ and I am watching your progress with much interest, vthor.  you have my 👍 and as you progress I’ll bind the lib to at least Dart.     

> __< p​lowsof:matrix.org >__ "Seraphis codebase"     

> __< ofrnxmr >__ Thanks for clearing up     

> __< k​ayabanerve:matrix.org >__ The Seraphis codebase isn't just the new cryptography though. The Seraphis codebase can be modified around the FCMP++ cryptography.     

> __< plowsof >__ so merging sneedlewoods      

> __< k​ayabanerve:matrix.org >__ So the cleaner transaction structure, the new wallet code, all of those fancy features like outgoing view keys?     

> __< k​ayabanerve:matrix.org >__ All of those are still eligible to be used from Seraphis and that work hasn't been wasted.     

> __< plowsof >__ as we're approaching the hour,   d. [Add CypherStack Carrot Spec Review](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/511)           

> __< r​ucknium:monero.social >__ I've thumbed-up sneedlewoods's proposal.     

> __< plowsof >__ which we definitely defer to MRL/MoneroTech :)      

> __< ofrnxmr >__ If i didnt thumb up, i meant to     

> __< s​needlewoods:monero.social >__ thanks for all the support everyone     

> __< plowsof >__ i thumb down for slave labour rates of course      

> __< ofrnxmr >__ Sneedle made the first ofrnmeme.      

> __< r​ucknium:monero.social >__ We had loose consensus at the last MRL meeting in favor of Cypher Stack doing a cryptographic review of the important security and privacy properties of the Carrot addressing scheme.     

> __< plowsof >__ n1ocs proxy vote wants to merge carrot     

> __< plowsof >__ nioCats*     

> __< ofrnxmr >__ kayabanerve, should we launch fcmp++ with or without carrot?     

> __< k​ayabanerve:matrix.org >__ Work on the new wallet API makes sense to me :) I personally decline to comment on the exact CCS above as I'm not sufficiently familiar, sorrym     

> __< vthor >__ " i thumb down for slave labour rates of course " <- this yes :D but my thumb up     

> __< plowsof >__ so this proposal already has the consensus for merge from MRL      

> __< k​ayabanerve:matrix.org >__ With. It isn't necessary but absolutely should be with.     

> __< plowsof >__ thanks for highlighting that Rucknium      

> __< plowsof >__ now on to   b. [Carrot animated video](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/506)         

> __< ofrnxmr >__ so goal is to have both complete within 11 months?     

> __< ofrnxmr >__ Comments need addressing     

> __< ofrnxmr >__ (re video)     

> __< plowsof >__ on a related note: vostoemisio seeks feedback for their introduction video on getmoneros front page reg the script. "https://pastebin.com/0tdY7aVc" was submitted by someone else (im not sure if publicly) which attempts to remove the 'fact's to be as timeless as possible. this is vostos proposed script https://pastebin.com/QJeK35gH      

> __< plowsof >__ geonic offered to fund the proposal , great to see :P     

> __< plowsof >__ i may have misinterpreted his comment https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/506#note_26652     

> __< ofrnxmr >__ 🤣     

> __< k​ayabanerve:matrix.org >__ No?     

> __< k​ayabanerve:matrix.org >__ All dev should be soft targeted for EOY.     

> __< p​lowsof:matrix.org >__ AOB? otherwise we ca end meeting and continue discussions     

> __< p​lowsof:matrix.org >__ can*     

> __< k​ayabanerve:matrix.org >__ 3 months, not 11.     

> __< ofrnxmr >__ ok even better     

> __< ofrnxmr >__ @rucknium, told ya     

> __< k​ayabanerve:matrix.org >__ I'd be upset and disappointed if we didn't have a testnet with FCMP++ and Carrot by February. If Carrot was a delay, I'd propose kicking it.     

> __< ofrnxmr >__ And how long do you presume until we can get s testnet up:     

> __< ofrnxmr >__ Damn, always ahead of me     

> __< ofrnxmr >__ Thanks for answering my qs b4 i ask them sir     

> __< k​ayabanerve:matrix.org >__ I've prior stated I'm not willing to let Carrot delay FCMP++.     

> __< p​lowsof:matrix.org >__ great place to end, thanks all for attending     

> __< k​ayabanerve:matrix.org >__ December, the soft target, if all goes well.     

> __< s​needlewoods:monero.social >__ thanks for the meeting, cu around     

> __< ofrnxmr >__ Coming to monerotopia?     

> __< k​ayabanerve:matrix.org >__ cc jeffro256     

> __< ofrnxmr >__ Oh     

> __< ofrnxmr >__ 1 more thing     

> __< ofrnxmr >__ Deverick's magic proposal for btcpayserver     

> __< ofrnxmr >__ And the moneronodo project is using @siren's moneropay     

> __< r​ucknium:monero.social >__ Hm. You did tell me. Anyway, I will wait until FCMP is truly testnet ready since I don't want to deal with chain halts, forks, etc. on stressnet, at least anymore than necessary.     

> __< ofrnxmr >__ I'll be happy to run the first testnets. Hopefully things go according to plan     

> __< r​ucknium:monero.social >__ Yes it's this one "Monerotopia Buildathon 2024" to improve Monero's integration in BTCPay:  https://github.com/MAGICGrants/Monero-Fund/issues/37     

> __< r​ucknium:monero.social >__ Community feedback is appreciated :)     

> __< r​ucknium:monero.social >__ I remember when I caused a consensus-critical integer underflow on the Townforge testnet and halted the chain :D     

> __< ofrnxmr >__ Noice. Thanks everyone      

> __< k​ayabanerve:matrix.org >__ I'll clarify my timeline for dev isn't my timeline for audits.     

> __< s​yntheticbird:monero.social >__ you welcome ofrnxmr     

> __< geonic >__ <plowsof> “on a related note: vostoemisio seeks feedback for their introduction video on getmoneros front page reg the script.” <- I gave feedback, haven’t heard back     

> __< r​ottenwheel:kernal.eu >__ ICYMI, interesting one... plowsof @plowsof:matrix.org https://forum.monero.space/t/translating-the-monero-wikipedia-article-into-new-languages/1800     

> __< v​ostoemisio:matrix.org >__ geonic:  I noted your feedback, and I agree that parts of our script are too technical. I think the best way to move forward is to use the original, existing script with updated facts. There’s nothing wrong with it, and it’s probably what the majority of the community would prefer.     

> __< plowsof >__ rottenwheel the OP suggests we submit machine translations to wikipedia to help. Johannes89 will be furious      

> __< s​yntheticbird:monero.social >__ y machien translations. You mean AI ?     

> __< s​yntheticbird:monero.social >__ s/y machien/by machine     

> __< r​ottenwheel:kernal.eu >__ Lol.     

> __< plowsof >__ evil wikipedia admin blocks my legitimiate translations. i need your help. can you please "With the help of various AI tools such as Chat GPT and Gemini, anyone can translate. Make sure that you double-check every section or sentence on Google Translate."     

> __< plowsof >__ legitimate*     


Automated by [this](https://github.com/plowsof/post-libera-meeting-logs)
